environments/backfillEvents.toml

```diff
@@ -0,0 +1,247 @@
+[exporter]
```

Comment:
I wonder if it's possible to use the `docker.toml` file instead of creating a new one, even though `docker.toml` has config for contracts that we don't necessarily need for this command. 🤔

Reply:
I think that could work, and may even be preferable for reduced complexity right now (since this config includes the lowest deployment block in `docker.toml`). The only thing that had me thinking we might want to go this route was the possibility that in the future we would only need to backfill events for transformers that had been deployed pretty recently (e.g. new collateral types). In that case, pointing at `docker.toml` would mean unnecessarily rechecking all earlier blocks for all watched events (since the starting block for backfilling is derived from the lowest deployment block in the config) - and this one would let us target only more recent blocks since the new collateral contracts were deployed. Think it's worth kicking the can to a later story when we cross that bridge?

Reply:
That's a good point that we'll likely want to backfill with a more recent block number for some transformers. 🤔  I'm good with leaving this as is, or kicking the decision down the road. 👍